apps/condo/pages/incident/[id]/index.tsx

```diff
@@ -98,7 +92,7 @@ const IncidentPropertiesField: React.FC<IncidentFieldProps> = ({ incident }) =>
                             </>
                         ) : (
                             <Typography.Link
-                                href={`/property/${get(property, 'id')}`}
+                                href={`/property/${property?.id}`}
```

Comment:
Maybe should use
```suggestion
                                href={`/property/${property?.id || '#'}`}
```

or

```suggestion
                                href={`/property/${property?.id || ''}`}
```
?